src/smart-components/access-management/UsersTable.tsx

```diff
@@ -113,6 +113,9 @@ const UsersTable: React.FunctionComponent<UsersTableProps> = ({ onAddUserClick }
           props: { isActionCell: true },
         },
       ],
+      props: {
+        onClick: () => onFocusUser && onFocusUser(user),
+      },
```

Comment:
this is probably not enough.
As mentioned in the DW extension's example `isClickable`, `onRowClick` and `isRowSelected` should be passed
https://react-data-view-pr-data-view-14.surge.sh/extensions/data-view/events-context 

Now, the drawer does not hide when you click the row again and the opened row is not focused when clicked. Also the cursor is not a pointer

Reply:
updated to use the events-context! :+1: